src/app/app-routing.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { CoursesComponent } from './courses/courses.component';
+
+const routes: Routes = [
+  { path: 'courses', component: CoursesComponent },
```

Comment:
`courses` string must be a constant or part of enum `AppRoutePath`

Reply:
done